interfaces/Portalicious/src/app/pages/project-payments/project-payments.page.ts

```diff
@@ -31,4 +37,15 @@ export class ProjectPaymentsPageComponent {
       detail: 'Functionality not implemented yet',
     });
   }
+  private paymentsQuery = injectQuery(
```

Comment:
```suggestion
  payments = injectQuery(
```

I wouldn't deviate from the default way we have defined the results of the queries. Also because using `private` here prevents you from accessing the `isPending()` in the template.